### PR TITLE
Fixed issue by opening cursor to metadata table & updating counts

### LINKED ISSUE #55: edit bulk_insert.cpp to insert nNodes and nEdges into the metadata table
In our graph implementations, we require that ```node_count``` and ```edge_count``` (defined in common.h) be present in the metadata table. Currently, this is missing. We should insert it right after inserting the nodes and edges in the bulk insertion step

To do this:
1. Open a cursor to the metadata table of each type of connection (```WT_CONNECTION *conn_std, *conn_adj, *conn_ekey;``` @ line 20 of bulk_insert.cpp).
2. insert nEdges = edge_times->num_inserted and nNodes = node_times->num_inserted into the metadata table.

https://github.com/ubc-systopia/margraphita/blob/88caae6102087c97eb3d28bb11350457da03875f/benchmark/bulk_insert.cpp#L458

### PR DESCRIPTION
**Pull request by apricot2012**
Fixes #55

### CODE REVIEW

File: benchmark/bulk_insert.cpp [hunk 1]
```diff
@@ -75,6 +75,47 @@ void print_time_csvline(std::string db_name,
...
+void insert_stats_to_session(WT_SESSION *session, int edgeNo, int nodeNo)
+{
+    WT_CURSOR *cursor;
+    if (int ret = session->open_cursor(
+                      session, "table:metadata", NULL, NULL, &cursor) != 0)
+    {
+        std::cout << "Failed to open metadata table";
+    }
+    cursor->set_key(cursor, node_count.c_str());
+    cursor->set_value(cursor, std::to_string(nodeNo).c_str());
```
Comment (artorhem) [inline benchmark/bulk_insert.cpp:87]: I think we should use ```sprintf``` or ```itoa``` to convert the ```int``` to ```char*```

```itoa``` will only work for 32-bit ints. sprintf works for any size ints as long as the buffer is large enough.

Comment (apricot2012) [reply benchmark/bulk_insert.cpp:87]: How long should I set the buffer to be? Should I create a new constant in common.h?

**Comment by apricot2012** [comment]
Fixed according to comments @artorhem

**Comment by artorhem** [comment]
Manually merged into iterators branch

**Pull request closed without merging**